src/node_api.cc

```diff
@@ -898,7 +898,8 @@ const char* error_messages[] = {nullptr,
                                 "Unknown failure",
                                 "An exception is pending",
                                 "The async work item was cancelled",
-                                "napi_escape_handle already called on scope"};
+                                "napi_escape_handle already called on scope",
+                                "invalid handle scope usage"};
```

Comment:
Capitalize "invalid" in accordance with the rest of `error_messages`.

The one immediately before this one is an exception, as `napi_escape_handle` is the name of a function, and function names are case-sensitive.